lib/locale/pl_PL/strings.php

```diff
@@ -2600,6 +2600,9 @@
 $_LANG['VLAN Description'] = 'Opis VLAN-u';
 $_LANG['VLAN ID'] = 'Identyfikator VLAN-u';
 $_LANG['Add VLAN'] = 'Dodaj VLAN';
+$_LANG['Assigned Network node'] = 'Przypisany węzeł sieciowy';
+$_LANG['Network Node'] = 'Węzeł sieciowy';
+$_LANG['Network Node ID'] = 'Nr Węzeła sieciowego';
```

Comment:
Panie ;)
**Nr węzła sieciowego**

Nie wystarczy miejsca na całe słowo numer? Trzeba skrótem? To kolumna w tabeli?

Reply:
Szacunek za napisanie `Nr` bez kropki :D

Reply:
:D
_Węzeła_ zrobiło mi dzień. No offense, śmiechłem z tego.

Reply:
@awbnet jak wg Ciebie powinno być docelowo:
a) $_LANG['Network Node identifier'] = 'Numer węzła sieciowego';
b) $_LANG['Network Node ID'] = 'ID węzła sieciowego';
c) $_LANG['Network Node ID'] = 'ID węzła sieciowego';
d) jeszcze inaczej ....... wpisz tu jak ...... 
?

Reply:
@interduo A to nie trzeba tylko węzęła poprawić? Myślałem, że reszta jest OK.

Reply:
> Węzeła już wyłapałem :D

> Szacunek za napisanie Nr bez kropki :D

O dziwo tylko w mianowniku ma być kropka. Człowiek się uczy całe życie...

Reply:
@awbnet zmieści się i cały wyraz ale jakoś w wielu miejscach się posługujemy takim skrótem... literówkę poprawiłem.

Reply:
@interduo OK, może kiedyś się zamieni.